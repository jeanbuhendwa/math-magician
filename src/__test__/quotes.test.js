import React from 'react';
import { render } from '@testing-library/react';
import RandomQuote from '../components/quotes';

describe('RandomQuote component', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<RandomQuote />);
    expect(asFragment()).toMatchSnapshot();
  });
});
