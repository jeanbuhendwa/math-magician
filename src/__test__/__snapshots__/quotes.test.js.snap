// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RandomQuote component should render correctly 1`] = `
<DocumentFragment>
  <div
    class="quoteLoading"
  >
    <img
      alt="Loading Background"
      src="loading-bg.svg"
    />
    <p>
      Quote Loading...
    </p>
  </div>
</DocumentFragment>
`;
